examples/hands_on_opencl/ex06/matmul.jl

```diff
@@ -91,15 +91,15 @@ function seq_mat_mul_sdot(Mdim::Int, Ndim::Int, Pdim::Int,
     end
 end
 
-info("=== Julia, matix mult (dot prod), order $ORDER ===")
+@info("=== Julia, matix mult (dot prod), order $ORDER ===")
 
 # force compilation
 seq_mat_mul_sdot(Mdim, Ndim, Pdim, h_A, h_B, h_C)
 
 for i in 1:COUNT
     fill!(h_C, 0.0)
     t1 = time()
-    #seq_mat_mul_sdot(Mdim, Ndim, Pdim, h_A, h_B, h_C)
+    seq_mat_mul_sdot(Mdim, Ndim, Pdim, h_A, h_B, h_C)
```

Comment:
Do you know why this was commented out previously? There doesn't seem to be any information in the PR that made this change (#11), so I just want to confirm that you are confident this change is correct :slightly_smiling_face: